go/tasks/pluginmachinery/utils/template_test.go

```diff
@@ -338,4 +342,18 @@ func TestReplaceTemplateCommandArgs(t *testing.T) {
 			"output/blah",
 		}, actual)
 	})
+
+	t.Run("sub raw output data prefix", func(t *testing.T) {
```

Comment:
can we create a test, where we are replacing other things with raw-output prefix?

Reply:
mmmm... yeah.  sorry does not compute.  can you elaborate?

Reply:
done